packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -166,6 +172,9 @@ class MessageList extends Widget<Properties> {
     const messageGroup = this._createComponent($messageGroup, MessageGroup, {
       items,
       alignment: this._messageGroupAlignment(userId),
+      showAvatar: this.option().showAvatar,
+      showUsername: this.option().showUsername,
+      showMessageTimestamp: this.option().showMessageTimestamp,
```

Comment:
[copypaste]
It's better to call `option` single time and use destructuring to make the code clearer

Reply:
4af31f